### DISCUSSION TITLE
Docs for mayaUSDExport

### DISCUSSION THREAD

**Discussion by andrewkaufman** [Autodesk/maya-usd#1032] [Q&A] [answered]
Am I correct in thinking [Pixar's usdExport doc](https://github.com/Autodesk/maya-usd/blob/595df7fea82c4d282d63cb7065962b4ed7d2dede/plugin/pxr/doc/README.md#usdexport) is the best available documentation for `maya.cmds.mayaUSDExport`? If so it might be helpful to move it up a few levels in the repo (and update any missing/stale flags). If not, can someone point me to the right docs?

**Comment by fowlertADSK** [comment, marked as answer]
You are correct that the Pixar documentation is currently the best available documentation for the import/export commands.

We started the process of creating a common set of commands that could be used by any plugin, but it is almost entirely based on the Pixar implementation so their documentation should still match what is available.  And like you suggest, we are planning on updating the doc’s and choosing a better location for it.  That work is in-progress and should be available soon.
